src/main/resources/com/google/api/codegen/nodejs/standalone_sample.snip

```diff
@@ -41,33 +41,41 @@
 #
 # FIXME: Replace the following function calls with calls to functions that emit full standalone samples. These stubs have been adapted from method_sample.snip
 @snippet standaloneSample(apiMethod, sample)
-  @switch sample.callingForm
-  @case "Request"
-    @if apiMethod.hasReturnValue
-      {@methodCallSampleCodeWithReturnValue(apiMethod, sample)}
-    @else
-      {@methodCallSampleCodeWithoutReturnValue(apiMethod, sample)}
+  async function {@sample.sampleFunctionName}({@formalArgs(sample.sampleInitCode.argDefaultParams)}) {
```

Comment:
Note: using `async` makes the code incompatible with Node.js v6. It might be perfectly fine since it's going to reach its EOL this April, but just wanted to point that out.

Reply:
But why is this function `async` at all? There are no `await` calls inside.